### ISSUE TITLE
Allow download of uploaded documents in user-chats

### ISSUE THREAD

**Issue by dhoegerlETM** [OpenAssistantGPT/OpenAssistantGPT#491]
In succession of #348 we not only want to download the chat history but also the uploaded documents from user-conversations. As the OpenAI API does not allow this natively, this very likely means that you must store the files additionally (at least for some weeks/months).

**Comment by marcolivierbouch** [comment]
![image](https://github.com/user-attachments/assets/8cb6db6c-8f18-46fc-bc80-b62906c7da17)

We just released a feature that allows you to display OpenAI file references in chat. You can look at our changelog [here ](https://www.openassistantgpt.io/changelog).

Does it fix your issue?

**Comment by dhoegerlETM** [comment]
@marcolivierbouch sorry I cannot see this new setting. But to state it clearly: we (as the chat-assistant provider) request to collect or download the uploaded files of our users. The users themselves only need to download the generated files from OpenAI's API.

**Comment by marcolivierbouch** [comment]
You want to export also files when you export your chat history?

**Comment by dhoegerlETM** [comment]
Exactly, either together with chat history or as own/explicit export would be great.
